views/login.html

```diff
@@ -0,0 +1,27 @@
+{% extends "templates/master.html" %}
+
+{% block pageTitle %}
+    Login
+{% endblock %}
+
+{% block content %}
+<div class="container">
+    {% if errormessage %}
+    <div class="alert alert-danger" role="alert">
+        {{ errormessage }}
```

Comment:
Make sure error messages are user friendly unlike this one
<img width="1226" alt="Screenshot 2023-09-13 at 13 18 02" src="https://github.com/anthony-raybould/bham1-front-end/assets/8750472/f53d2427-f950-434c-8b3b-454b77aeedef">